docs/how-to-setup-wsl.md

```diff
@@ -1,5 +1,12 @@
 # Setting up WSL for Working with freeCodeCamp Locally
 
+> [!NOTE]
+> Requirements:
+> 
+> **WSL 1**: Windows 10 64-bit, version 1607
+> 
+> **WSL 2**: Windows 10 64-bit, version 2004, Build 19041 or higher (As of June, only available once part of the Windows Insider Program)
```

Comment:
> `(As of June, only available once part of the Windows Insider Program)`

Can you confirm this is correct? I believe its available globally. I think I was able to install it on Windows 10 normally without having signed up for the insider program.

Reply:
Right, it should be the other way around. The version was released on the 27/05

Reply:
Did you get a chance to remove the line? I think this is good to go pending the change.